api/http/management_external_test.go

```diff
@@ -80,6 +83,7 @@ func TestIOTHubExternal(t *testing.T) {
 		t.Skip("test.device-id is not provided nor valid")
 		return
 	}
+	t.Skip("FIXME: Uses proxy API")
```

Comment:
What does "uses proxy API" mean?
I fixed this external test in my PR; I would suggest not touching it in your PR.

Reply:
I was just lazily disabled the test since it requires your PR. I'll revert this diff.